gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -181,19 +192,107 @@ public void cleanUp() throws Exception {
   }
 
   @Test
-  public void testCreate() throws Exception {
+  public void testRestart() throws Exception {
+    Map<String, String> flowProperties = Maps.newHashMap();
+    flowProperties.put("param1", "value1");
+    flowProperties.put(ServiceConfigKeys.FLOW_SOURCE_IDENTIFIER_KEY, TEST_SOURCE_NAME);
+    flowProperties.put(ServiceConfigKeys.FLOW_DESTINATION_IDENTIFIER_KEY, TEST_SINK_NAME);
+    FlowConfig flowConfig = new FlowConfig().setId(new FlowId().setFlowGroup(TEST_GROUP_NAME).setFlowName(MockedSpecCompiler.UNCOMPILABLE_FLOW))
```

Comment:
If this flow is Compiled correctly, then can we call this something other than a UNCOMPILABLE_FLOW, as that is confusing?

Reply:
the flow is not compilable, hence the name